app/templates/components/team-picker.hbs

```diff
@@ -3,7 +3,7 @@
   <div class="team left-team" {{action "pick" left.ref}}>
```

Comment:
seems like I could make this a nice component. need to arrange a good mental model of how actions would be impacted here.